docs/workstation_release_management.rst

```diff
@@ -28,48 +28,66 @@ On release day (usually at least a couple weeks after releasing your first rc pa
 Step 1: Create a release candidate (rc) tag
 -------------------------------------------
 
-1. Create a release branch in the repo of the component you want to release.
-2. Push a changelog commit.
-3. Push an rc tag in the format ``<major>.<minor>.<patch>~rcN`` on your new commit. We will be building from this tag in the next step.
+1. Create a release branch (eg. ``release/1.2.3``) in the repo of the component you want to release.
+2. Update the component version to the intended release version (not the RC!) using the ``update_version.sh`` script.
+3. Update the changelog to include changes since the last release.
+4. Commit and push the version and changelog update.
+5. Push an rc tag in the format ``<major>.<minor>.<patch>-rcN`` on your new commit. We will be building from this tag in the next step.
 
 Step 2: Build and deploy the package to ``apt-test``
 ----------------------------------------------------
 
-1. Open a terminal in your named DispVM called ``sd-dev-dvm`` (see :ref:`How to create the DispVM for building packages`).
+1. Open a terminal in your ``sd-dev-dvm`` builder DispVM (see :ref:`How to create the DispVM for building packages`), and prepare to record terminal output- either by using a utility like ``script``, or just by making sure you have infinite scrollback enabled to allow for cut/pasting later.
+2. Clone the the ``build-logs`` and ``securedrop-apt-test`` repos.
 
-2. Clone ``securedrop-builder`` and install its dependencies (https://github.com/freedomofpress/securedrop-builder/tree/HEAD/workstation-bootstrap/wheels):
+3. Clone ``securedrop-builder`` and install its dependencies (https://github.com/freedomofpress/securedrop-builder/tree/HEAD/workstation-bootstrap/wheels):
 
   .. code-block:: sh
 
    git clone git@github.com:freedomofpress/securedrop-builder.git
    cd securedrop-builder
    make install-deps  # This also confifgures the git-lfs repo used to store SecureDrop Workstation dependencies
 
-3. Create a Debian changelog entry for the new version of the package you are about to build.
+4. Create a Debian changelog entry for the new version of the package you are about to build, using the format ``x.y.z~rcN`` - note the ``~``, as this differs from the git tag format.
```

Comment:
The script takes care of the `s/-/~/g` now, so I think we can keep the documentation simple and have humans only input the version in one format and let scripts munge it as needed.